[version: 1.0.12, 2023-07-05T18:12:02.507Z, pid: 45073] > /bin/python -m dvc root - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:02.531Z, pid: 45073] > /bin/python -m dvc root - FAILED with code 1 (32ms)
/bin/python: No module named dvc

[version: 1.0.12, 2023-07-05T18:12:02.540Z, pid: 45075] > git rev-parse --show-toplevel - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:02.549Z, pid: 45076] > /bin/python -m dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:02.552Z, pid: 45075] > git rev-parse --show-toplevel - FAILED with code 128 (19ms)
fatal: not a git repository (or any parent up to mount point /mnt)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).

[version: 1.0.12, 2023-07-05T18:12:02.562Z, pid: 45077] > git version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:02.565Z, pid: 45076] > /bin/python -m dvc --version - FAILED with code 1 (24ms)
/bin/python: No module named dvc

[version: 1.0.12, 2023-07-05T18:12:02.573Z, pid: 0] > dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:02.575Z, pid: 45077] > git version - COMPLETED (22ms)
[version: 1.0.12, 2023-07-05T18:12:02.584Z, pid: 0] > dvc --version - FAILED (19ms)
Command failed with ENOENT: dvc --version
spawn dvc ENOENT
[version: 1.0.12, 2023-07-05T18:12:02.594Z, pid: 45080] > git rev-parse --show-toplevel - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:02.601Z, pid: 45080] > git rev-parse --show-toplevel - FAILED with code 128 (15ms)
fatal: not a git repository (or any parent up to mount point /mnt)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).

[version: 1.0.12, 2023-07-05T18:12:02.610Z, pid: 45081] > git rev-parse --show-toplevel - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:02.621Z, pid: 45081] > git rev-parse --show-toplevel - FAILED with code 128 (18ms)
fatal: not a git repository (or any parent up to mount point /mnt)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).

[version: 1.0.12, 2023-07-05T18:12:07.607Z, pid: 45086] > /bin/python -m dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:07.620Z, pid: 45086] > /bin/python -m dvc --version - FAILED with code 1 (32ms)
/bin/python: No module named dvc

[version: 1.0.12, 2023-07-05T18:12:07.628Z, pid: 0] > dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:07.629Z, pid: 0] > dvc --version - FAILED (8ms)
Command failed with ENOENT: dvc --version
spawn dvc ENOENT
[version: 1.0.12, 2023-07-05T18:12:12.650Z, pid: 45088] > /bin/python -m dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:12.664Z, pid: 45088] > /bin/python -m dvc --version - FAILED with code 1 (33ms)
/bin/python: No module named dvc

[version: 1.0.12, 2023-07-05T18:12:12.671Z, pid: 0] > dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:12.673Z, pid: 0] > dvc --version - FAILED (9ms)
Command failed with ENOENT: dvc --version
spawn dvc ENOENT
[version: 1.0.12, 2023-07-05T18:12:17.700Z, pid: 45096] > /bin/python -m dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:17.739Z, pid: 45096] > /bin/python -m dvc --version - FAILED with code 1 (64ms)
/bin/python: No module named dvc

[version: 1.0.12, 2023-07-05T18:12:17.749Z, pid: 0] > dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:17.751Z, pid: 0] > dvc --version - FAILED (10ms)
Command failed with ENOENT: dvc --version
spawn dvc ENOENT
[version: 1.0.12, 2023-07-05T18:12:22.763Z, pid: 45104] > /bin/python -m dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:22.777Z, pid: 45104] > /bin/python -m dvc --version - FAILED with code 1 (24ms)
/bin/python: No module named dvc

[version: 1.0.12, 2023-07-05T18:12:22.786Z, pid: 0] > dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:22.787Z, pid: 0] > dvc --version - FAILED (9ms)
Command failed with ENOENT: dvc --version
spawn dvc ENOENT
[version: 1.0.12, 2023-07-05T18:12:27.797Z, pid: 45109] > /bin/python -m dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:27.811Z, pid: 45109] > /bin/python -m dvc --version - FAILED with code 1 (22ms)
/bin/python: No module named dvc

[version: 1.0.12, 2023-07-05T18:12:27.817Z, pid: 0] > dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:27.819Z, pid: 0] > dvc --version - FAILED (7ms)
Command failed with ENOENT: dvc --version
spawn dvc ENOENT
[version: 1.0.12, 2023-07-05T18:12:32.830Z, pid: 45118] > /bin/python -m dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:32.843Z, pid: 45118] > /bin/python -m dvc --version - FAILED with code 1 (22ms)
/bin/python: No module named dvc

[version: 1.0.12, 2023-07-05T18:12:32.850Z, pid: 0] > dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:32.852Z, pid: 0] > dvc --version - FAILED (8ms)
Command failed with ENOENT: dvc --version
spawn dvc ENOENT
[version: 1.0.12, 2023-07-05T18:12:37.862Z, pid: 45123] > /bin/python -m dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:37.881Z, pid: 45123] > /bin/python -m dvc --version - FAILED with code 1 (27ms)
/bin/python: No module named dvc

[version: 1.0.12, 2023-07-05T18:12:37.890Z, pid: 0] > dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:37.892Z, pid: 0] > dvc --version - FAILED (10ms)
Command failed with ENOENT: dvc --version
spawn dvc ENOENT
[version: 1.0.12, 2023-07-05T18:12:42.901Z, pid: 45172] > /bin/python -m dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:42.915Z, pid: 45172] > /bin/python -m dvc --version - FAILED with code 1 (21ms)
/bin/python: No module named dvc

[version: 1.0.12, 2023-07-05T18:12:42.923Z, pid: 0] > dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:42.924Z, pid: 0] > dvc --version - FAILED (8ms)
Command failed with ENOENT: dvc --version
spawn dvc ENOENT
[version: 1.0.12, 2023-07-05T18:12:47.942Z, pid: 45174] > /bin/python -m dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:47.956Z, pid: 45174] > /bin/python -m dvc --version - FAILED with code 1 (30ms)
/bin/python: No module named dvc

[version: 1.0.12, 2023-07-05T18:12:47.962Z, pid: 0] > dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:47.967Z, pid: 0] > dvc --version - FAILED (11ms)
Command failed with ENOENT: dvc --version
spawn dvc ENOENT
[version: 1.0.12, 2023-07-05T18:12:52.977Z, pid: 45185] > /bin/python -m dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:52.993Z, pid: 45185] > /bin/python -m dvc --version - FAILED with code 1 (24ms)
/bin/python: No module named dvc

[version: 1.0.12, 2023-07-05T18:12:53.000Z, pid: 0] > dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:53.004Z, pid: 0] > dvc --version - FAILED (11ms)
Command failed with ENOENT: dvc --version
spawn dvc ENOENT
[version: 1.0.12, 2023-07-05T18:12:58.028Z, pid: 45188] > /bin/python -m dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:58.068Z, pid: 45188] > /bin/python -m dvc --version - FAILED with code 1 (62ms)
/bin/python: No module named dvc

[version: 1.0.12, 2023-07-05T18:12:58.076Z, pid: 0] > dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:12:58.078Z, pid: 0] > dvc --version - FAILED (9ms)
Command failed with ENOENT: dvc --version
spawn dvc ENOENT
[version: 1.0.12, 2023-07-05T18:13:03.091Z, pid: 45193] > /bin/python -m dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:13:03.104Z, pid: 45193] > /bin/python -m dvc --version - FAILED with code 1 (22ms)
/bin/python: No module named dvc

[version: 1.0.12, 2023-07-05T18:13:03.111Z, pid: 0] > dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:13:03.112Z, pid: 0] > dvc --version - FAILED (7ms)
Command failed with ENOENT: dvc --version
spawn dvc ENOENT
[version: 1.0.12, 2023-07-05T18:13:08.123Z, pid: 45199] > /bin/python -m dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:13:08.136Z, pid: 45199] > /bin/python -m dvc --version - FAILED with code 1 (22ms)
/bin/python: No module named dvc

[version: 1.0.12, 2023-07-05T18:13:08.143Z, pid: 0] > dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:13:08.144Z, pid: 0] > dvc --version - FAILED (7ms)
Command failed with ENOENT: dvc --version
spawn dvc ENOENT
[version: 1.0.12, 2023-07-05T18:13:13.156Z, pid: 45207] > /bin/python -m dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:13:13.169Z, pid: 45207] > /bin/python -m dvc --version - FAILED with code 1 (22ms)
/bin/python: No module named dvc

[version: 1.0.12, 2023-07-05T18:13:13.176Z, pid: 0] > dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:13:13.177Z, pid: 0] > dvc --version - FAILED (7ms)
Command failed with ENOENT: dvc --version
spawn dvc ENOENT
[version: 1.0.12, 2023-07-05T18:13:18.189Z, pid: 45218] > /bin/python -m dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:13:18.203Z, pid: 45218] > /bin/python -m dvc --version - FAILED with code 1 (24ms)
/bin/python: No module named dvc

[version: 1.0.12, 2023-07-05T18:13:18.210Z, pid: 0] > dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:13:18.211Z, pid: 0] > dvc --version - FAILED (7ms)
Command failed with ENOENT: dvc --version
spawn dvc ENOENT
[version: 1.0.12, 2023-07-05T18:13:23.235Z, pid: 45225] > /bin/python -m dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:13:23.277Z, pid: 45225] > /bin/python -m dvc --version - FAILED with code 1 (63ms)
/bin/python: No module named dvc

[version: 1.0.12, 2023-07-05T18:13:23.285Z, pid: 0] > dvc --version - INITIALIZED
[version: 1.0.12, 2023-07-05T18:13:23.289Z, pid: 0] > dvc --version - FAILED (11ms)
Command failed with ENOENT: dvc --version
spawn dvc ENOENT
